hips/draw/simple.py

```diff
@@ -61,7 +61,7 @@ class HipsPainter:
     (1000, 2000)
     """
 
-    def __init__(self, geometry: WCSGeometry, hips_survey: HipsSurveyProperties, tile_format: str) -> None:
+    def __init__(self, geometry: WCSGeometry, hips_survey: Union[str, 'HipsSurveyProperties'], tile_format: str) -> None:
```

Comment:
The common thing to do is to put the type, and only to put quotes if there's a forward reference issue.
So please remove the quotes from `HipsSurveyProperties` here in the type annotation, and review the diff of this PR on Github to fix the other cases (I see one below).